src/util-thash.c

```diff
@@ -334,6 +334,11 @@ THashTableContext *THashInit(const char *cnf_prefix, size_t data_size,
     return ctx;
 }
 
+void THashUpdateMemuse(THashTableContext *ctx, const size_t data_size)
```

Comment:
Should use a name that suggests adding or incrementing here.  Probably also need to track decrementing as well.

Reply:
Indeed. That's a big mistake.